### PR TITLE
Expose NavigationPropTypes via NavigationExperimental.PropTypes

### PR DESCRIPTION
**Pull request by nevir**
Useful for building components that compose navigation components

### CODE REVIEW

**Comment by hedgerwang** [comment]
@nevir : were you trying to build an example or a real project?

I wonder why not just call `const NavigationPropTypes = require('NavigationPropTypes');` directly from your own project.

**Comment by nevir** [comment]
AFAIK you can't require `@providesModule`'d modules from outside of your project sources (e.g. I can't reach those types from a `npm` package)

**Comment by satya164** [comment]
@hedgerwang Haste modules are considered just an implementation detail of RN and we discourage their usage outside of RN.

**Comment by satya164** [comment]
@hedgerwang Should we merge this or reject?

**Comment by hedgerwang** [comment]
@facebook-github-bot shipit

**Comment by hedgerwang** [comment]
@satya164 PR looks good to me. Let's wait for builds to be good then ship this.

**Comment by satya164** [comment]
@hedgerwang Tests seem to be passing now.

@facebook-github-bot shipit

File: Libraries/NavigationExperimental/NavigationExperimental.js [hunk 1]
```diff
@@ -40,6 +41,9 @@ const NavigationExperimental = {
   CardStack: NavigationCardStack,
   Header: NavigationHeader,
   LegacyNavigator: NavigationLegacyNavigator,
+
+  // Utils
```
Comment (mkonicek) [inline Libraries/NavigationExperimental/NavigationExperimental.js]: nit: Can you remove the comment "Utils" here? I find the code self explanatory.

**Comment by mkonicek** [comment]
@nevir Are the Flow errors related to this pull request? https://travis-ci.org/facebook/react-native/jobs/124851856

iOS Travis tests are broken now but JS tests are green. Can you rebase this pull request so we can see test results please?

**Comment by nevir** [comment]
@mkonicek - yup, rebased.

As best I can tell, the test failures were unrelated

**Comment by facebook-github-bot-4** [comment]
@ericvicenti would you mind taking a look at this pull request? It's been a while since the last commit was reviewed.

**Comment by satya164** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**